psi4/driver/aliases.py

```diff
@@ -102,21 +102,25 @@ def sherrill_gold_standard(func, label, **kwargs):
     >>> optimize('sherrill_gold_standard', corl_basis='cc-pV[DT]Z', delta_basis='3-21g')
 
     """
-    kwargs['scf_basis'] = kwargs.get('scf_basis', 'aug-cc-pVQZ')
-    kwargs['scf_scheme'] = kwargs.get('scf_scheme', driver_cbs.xtpl_highest_1)
-
-    kwargs['corl_wfn'] = kwargs.get('corl_wfn', 'mp2')
-    kwargs['corl_basis'] = kwargs.get('corl_basis', 'aug-cc-pV[TQ]Z')
-    kwargs['corl_scheme'] = kwargs.get('corl_scheme', driver_cbs.corl_xtpl_helgaker_2)
-
-    kwargs['delta_wfn'] = kwargs.get('delta_wfn', 'ccsd(t)')
-    kwargs['delta_wfn_lesser'] = kwargs.get('delta_wfn_lesser', 'mp2')
-    kwargs['delta_basis'] = kwargs.get('delta_basis', 'aug-cc-pVTZ')
-    kwargs['delta_scheme'] = kwargs.get('delta_scheme', driver_cbs.xtpl_highest_1)
-
-    if label == 'custom_function':
-        label = 'Sherrill Group Gold Standard'
-    return driver_cbs.cbs(func, label, **kwargs)
+    scf = {
+        'wfn': 'hf',
+        'basis': kwargs.pop('scf_basis', 'aug-cc-pVQZ'),
+        'scheme': kwargs.pop('scf_scheme', 'xtpl_highest_1')
+    }
+    corl = {
+        'wfn': kwargs.pop('corl_wfn', 'mp2'),
+        'basis': kwargs.pop('corl_basis', 'aug-cc-pV[TQ]Z'),
+        'scheme': kwargs.pop('corl_scheme', 'corl_xtpl_helgaker_2')
+    }
+    delta = {
+        'wfn': kwargs.pop('delta_wfn', 'ccsd(t)'),
+        'wfn_lesser': kwargs.pop('delta_wfn_lesser', 'mp2'),
+        'basis': kwargs.pop('delta_basis', 'aug-cc-pVTZ'),
+        'scheme': kwargs.pop('delta_scheme', 'xtpl_highest_1')
+    }
+
+    kwargs["cbs_metadata"] = [scf, corl, delta]
+    return driver_cbs.cbs(func, "", **kwargs)
 
 
 def allen_focal_point(func, label, **kwargs):
```

Comment:
The test suite has zero coverage of `allen_focal_point`. I insist on adding a regression test on energies, so we can be sure this PR doesn't change them.

Reply:
yeah, trouble is it would scarcely ever be run since it depends on mrcc. which reminds me I ought to follow Peter's recc and at least check mrcc present.